### ISSUE TITLE
TDP-43 clumps in brain

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11599]
**Preferred term label:**
TDP-43 clumps in brain

**Synonyms**
TAR DNA-binding protein 43 proteinopathy

**Definition (free text, please give PubMed ID)**
https://pmc.ncbi.nlm.nih.gov/articles/PMC6536849/

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Limbic-predominant age-related TDP-43 encephalopathy (LATE)
Frontotemporal lobar degeneration with TDP43 inclusions 

**Your nano-attribution (ORCID)**
